src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicMethodExtractor.vb

```diff
@@ -81,11 +81,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractMethod
             Return CType(methodNames.FirstOrDefault(Function(t) t.Parent.Kind <> SyntaxKind.SubStatement AndAlso t.Parent.Kind <> SyntaxKind.FunctionStatement), SyntaxToken)
         End Function
 
-        Protected Overrides Async Function CheckTypeAsync(document As Document,
-                                               contextNode As SyntaxNode,
-                                               location As Location,
-                                               type As ITypeSymbol,
-                                               cancellationToken As CancellationToken) As Task(Of OperationStatus)
+        Protected Overrides Function CheckType(
```

Comment:
i think this logic is identical C#'s.  i will try to unify to one helper in a followup pr.